x/emissions/keeper/inference_synthesis/network_inference_builder.go

```diff
@@ -250,31 +251,37 @@ func (b *NetworkInferenceBuilder) calcOneInValue(oneInForecaster Worker) (allora
 	palette.ForecastImpliedInferenceByWorker = forecastImpliedInferencesWithForecaster
 	palette.Forecasters = []Worker{oneInForecaster}
 
+	regret, noPriorRegret, err := palette.K.GetOneInForecasterSelfNetworkRegret(palette.Ctx, palette.TopicId, oneInForecaster)
+	if err != nil {
+		return alloraMath.Dec{}, errorsmod.Wrapf(err, "Error getting one-in forecaster regret")
+	}
+
+	if noPriorRegret {
+		return alloraMath.NewNaN(), nil
+	}
+
+	palette.ForecasterRegrets[oneInForecaster] = &StatefulRegret{
+		regret:        regret.Value,
+		noPriorRegret: noPriorRegret,
```

Comment:
so this will always be false? Should we mark it as such by setting it to false?

Reply:
yeah, why you think it's better to set as false. I think it is understood that it always has to be false in this case.

Reply:
just more clarity, doesn't really matter though